Library/Formula/speech-tools.rb

```diff
@@ -0,0 +1,23 @@
+class SpeechTools < Formula
+  homepage "http://festvox.org/docs/speech_tools-2.4.0/"
+  url "http://festvox.org/packed/festival/2.4/speech_tools-2.4-release.tar.gz"
+  sha1 "5b0ebd39bb7afa33e4093724a2123bdc62a6aebc"
+
+  def install
+    ENV.deparallelize
+    system "./configure"
+    system "make"
```

Comment:
I tried your suggested syntax and a few variants, but it doesn't seem to work -- apparently the array consists of strings, not files, and my ruby is weak.

I'll just leave the block syntax as above.